app/hooks/useMouseMove.ts

```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+
+const useMouseMove = () => {
```

Comment:
```suggestion
const useMouseMove = (timeout: number = 5000) => {
```

Lets make the `5000` an argument